src/Sylius/Component/Core/Distributor/ProportionalIntegerDistributor.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Distributor;
```

Comment:
We don't depends on any class from `Core` component. IMHO we should put this class to promotion component, it could be useful to anybody, who is using it.